connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTest.java

```diff
@@ -153,6 +164,7 @@ public void setup() {
         workerProps.put("config.providers.file.class", MockFileConfigProvider.class.getName());
         mockFileProviderTestId = UUID.randomUUID().toString();
         workerProps.put("config.providers.file.param.testId", mockFileProviderTestId);
+        workerProps.put("group.id", GROUP_ID);
```

Comment:
This `group.id` property is not defined in `StandaloneConfig`. See my earlier comment.

Reply:
Removed group.id and related check.